addons/a11y/src/components/Tabs.tsx

```diff
@@ -77,7 +77,7 @@ const Item = styled.button(
 
 const TabsWrapper = styled.div({});
 
-const List = styled.div(({ theme }) => ({
+const List = styled.div<{}>(({ theme }) => ({
```

Comment:
What's the benefit of `style.div<{}>(...)`? It's a bit ugyly and it seems like there are other places in this PR where it's still written `style.div(...)` so I'm just wondering what's the rationale here?

Reply:
I wish i didn't have to:
https://github.com/emotion-js/emotion/issues/1298